client/components/Assignments/AssignmentEditor.tsx

```diff
@@ -297,15 +297,18 @@ export class AssignmentEditorComponent extends React.Component {
                         />
                     </Row>
                 ) : (
-                    <SelectUserInput
-                        field={this.FIELDS.USER}
-                        label={gettext('User')}
-                        value={this.state.user}
-                        onChange={this.onUserChange}
-                        users={this.state.filteredUsers}
-                        popupContainer={popupContainer}
-                        readOnly={disableUserSelection}
-                    />
+                    <Row style={{padding: '2rem 0', margin: '0 0 1.8em 0'}}>
+                        <SelectUser
+                            selectedUserId = {this.state.userId}
+                            onSelect={(user) => {
+                                this.onUserChange(null, user);
+                            }}
+                            autoFocus={false}
+                            horizontalSpacing={true}
+                            clearable={true}
+                            data-test-id= {this.FIELDS.USER}
```

Comment:
Component doesn't have `data-test-id` prop defined in the interface. It already has "select-user-dropdown" data-test-id that you can use in tests when needed.